pydy/viz/scene.py

```diff
@@ -459,8 +461,15 @@ def remove_static_html(self, force=False):
 
     def display(self):
         """Displays the scene in the default webbrowser."""
-        self.create_static_html()
-        run_server(scene_file=self._scene_json_file)
+        tmp_dir = tempfile.mkdtemp(prefix='pydy_', suffix='_viz')
+        src = os.path.join(os.path.dirname(__file__), 'static')
+        distutils.dir_util.copy_tree(src, tmp_dir)
+        shutil.move(os.path.join(os.getcwd(), self._scene_json_file),
+                    os.path.join(tmp_dir, self._scene_json_file))
+        shutil.move(os.path.join(os.getcwd(), self._simulation_json_file),
+                    os.path.join(tmp_dir, self._simulation_json_file))
```

Comment:
I'm not sure about this method. The `generate...json()` method dumps the files into the cwd. That method should be dumping them into this tmp directory.

With this current method, if you call `generate_visualization_json()` multiple times you will fill the cwd with json files and then if you call `display()` you'll only move the last json file to the tmp dir. So this doesn't really solve the problem.